tests/test_storage.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+from os import path
+
+from tg import expose, TGController, AppConfig
+from webtest import TestApp, Upload
+
+import tgext
+from tgext.utils.storage import store
+
+
+TESTS_DIR = path.abspath(path.join(path.dirname(tgext.__file__), '..', 'tests'))
+PATH_TO_FILE = path.join(TESTS_DIR, 'stuff', 'tg.png')
+PATH_TO_STORAGE = path.join(TESTS_DIR, 'storage')
```

Comment:
This should probably be https://docs.python.org/2/library/tempfile.html#tempfile.tempdir it's not a good practice to write into the source code directory itself while running tests, might lead to package in the release unwanted files.  (In this case it won't probably happen as tgext.utils does not distribute package resources, but in the future we might add some and forget that tests are writing new files)